flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/CreateTableASDesc.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.delegation.hive;
+
+import org.apache.hadoop.hive.ql.parse.ASTNode;
+
+import java.io.Serializable;
+
+/** A desc for CTAS. */
+public class CreateTableASDesc implements Serializable {
```

Comment:
There are quite a lot of `Desc`, and maybe more in the future. Could you move them into a separate package `desc`? That would be clearer in the package structure.